.gitignore

```diff
@@ -0,0 +1,138 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Not sure if it's needed to add `.gitignore` for this repo, just don't commit these files and remove `gitignore` itself